client/src/reducers/challengeStatReducers/challengeStatsReducer.js

```diff
@@ -0,0 +1,66 @@
+import {
+    FETCH_CHALLENGESTATS_REQUEST,
+    FETCH_CHALLENGESTATS_SUCCESS,
+    FETCH_CHALLENGESTATS_FAILURE,
+    ADD_CHALLENGESTAT,
+    DELETE_CHALLENGESTAT
+} from '../../constants/challengeStatConstants/ActionTypes'
+
+const initChallengeStatsState = {
+    data: [],
+    count: 0,
+    loading: false,
+    error: ''
+}
+
+const challengeStatsReducer = (state = initChallengeStatsState, action) => {
+    switch(action.type) {
+        case FETCH_CHALLENGESTATS_REQUEST:
+            return {
+                ...state,
+                loading: true
+            }
+
+        case FETCH_CHALLENGESTATS_SUCCESS:
+            console.log("reducer");
+
+            const challengeStats = action.payload.map(challengeStat => ({
+                name: challengeStat,
+                id: 1
```

Comment:
just asking... always 1? shouldn't this be an unique id per each challengeStat? 🤔

Reply:
Actually we don't use this component, so it will get removed! Thx for the observation.